web/src/styles/components/alerts.scss

```diff
@@ -2,7 +2,43 @@
   mask: url(/vendor/img/usa-icons/error.svg) no-repeat center !important;
 }
 
+.usa-alert--note::before {
+  content: url(/img/push-pin.svg);
+  height: 2.5rem;
+  width: 1.5rem;
+  display: block;
+  position: absolute;
+  left: 0.91rem;
+}
```

Comment:
So this isn't exactly how I would have wanted to do this but, we're using USWDS stuff for most of our alert types and this matches their patterns.